components/Version.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Make this component a general component and absorb Architectures.tsx and OperatingSystems.tsx into it. That way we have less to maintain. 

If you need to do item specific styling I like to pass in a BoxProp element that can be unpacked in the wrapper box so I can pad the element how I would like.

Reply:
Edited version component as Filters.tsx and moved other components there as well.